cli/src/wallet_cli/mod.rs

```diff
@@ -1363,14 +1362,7 @@ async fn print_wallet_address(wallet: &Wallet) -> Result<(), Error> {
 
     log = format!(
         "{log}\nOutputs: {:#?}\nBase coin amount: {}\nNative Tokens: {:?}\nAccounts: {:?}\nFoundries: {:?}\nNFTs: {:?}\nDelegations: {:?}\nAnchors: {:?}\n",
-        output_ids,
```

Comment:
maybe split this up or something ew

Reply:
Done

Reply:
Can't we just push the strings rather than the weird formatting thing?

Reply:
Or even just log them one at a time??

Reply:
pleasssee

Reply:
@thibault-martinez want to do this?